lib/install/validate-args.js

```diff
@@ -14,12 +14,23 @@ module.exports = function (idealTree, args, next) {
 
   asyncMap(args, function (pkg, done) {
     chain([
+      [hasMinimumFields, pkg],
       [checkSelf, idealTree, pkg, force],
       [isInstallable, pkg]
     ], done)
   }, next)
 }
 
+function hasMinimumFields (pkg, cb) {
+  if (pkg.name === '' || pkg.name == null) {
```

Comment:
maybe I'm being kinda golfy, but I'm also curious: Why not just `if (!pkg.name) {}`?